vagrant/ansible/roles/sit.storagescale/tasks/common/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Process OS specific tasks
+  include_tasks: "{{ include_file }}"
+  with_first_found:
+    - files: "{{ config.os.includes }}"
+  loop_control:
+    loop_var: include_file
+
+# - name: Disable rpcbind
```

Comment:
The Storage Scale documentation recommends to disable rpcbind, so I added it as a step. However CentOS8 doesn't have it installed by default, which causes the task to fail.

I wasn't sure if it was better to keep it for reference or remove it (or make it a bit more complex to detect if rpcbind is actually installed).

Probably I can just remove it. I'll do it in the next update.

Reply:
we can even add one more commented line explaining this situation which can be used in future, so that this recommendation by scale does not go out of our radar (if that sound good to you.)

Reply:
or we can ask ansible to ignore any failure. ie. handle both cases where rpcbind is installed or when not installed.